lua/bufferline/animate.lua

```diff
@@ -42,7 +42,11 @@ local function animate_tick(timer, state)
   -- Went overtime, stop the animation!
     state.running = false
     state.fn(state.final, state)
-    timer:stop()
+    if state.timer then
```

Comment:
Is the `local function stop` reusable here? i.e. `stop(state)`

Reply:
Check new commit.